### PR TITLE
Update runtime to 44

### LINKED ISSUE #18: Runtime org.gnome.Platform branch 42 is end-of-life
Reference: https://github.com/cvfosammmm/Setzer/issues/313.

Running `flatpak update` shows:

```
Info: runtime org.gnome.Platform branch 42 is end-of-life, with reason:
   The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   org.cvfosammmm.Setzer
```

Thanks for your hard work btw!

### PR DESCRIPTION
**Pull request by oscfdezdz**
Closes #18

### CODE REVIEW

**Comment by cvfosammmm** [comment]
Thanks a lot. Why did you put the webkit module in a separate file?

**Comment by oscfdezdz** [comment]
Despite using the WebKit version of runtime 42, it still doesn't work.

```
Traceback (most recent call last):
  File "/app/bin/setzer", line 29, in <module>
    from setzer.workspace.workspace import Workspace
  File "/app/lib/python3.10/site-packages/setzer/workspace/workspace.py", line 30, in <module>
    import setzer.workspace.help_panel.help_panel as help_panel
  File "/app/lib/python3.10/site-packages/setzer/workspace/help_panel/help_panel.py", line 20, in <module>
    gi.require_version('WebKit2', '4.0')
  File "/usr/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace WebKit2 not available for version 4.0
```

**Comment by oscfdezdz** [comment]
> Thanks a lot. Why did you put the webkit module in a separate file?

Since WebKit required its own modules, I thought it would be better to separate it, but I can place it all in the manifest if you like.

**Comment by hfiguiere** [comment]
that's because runtime 43 and 44 have version 4.1. In python usually just changing the requirement is enough. As the API breakage is the removal of libsoup-2.4 in favor of libsoup-3.0.

File: webkit.json
Comment (hfiguiere) [inline webkit.json:1]: you shouldn't need that.

**Comment by oscfdezdz** [comment]
Latest published version is broken as the runtime in master lacks WebKit2 4.1.

**Comment by hfiguiere** [comment]
so nobody ran #20 before merging it?

**Pull request merged**